### ISSUE TITLE
Workflow failure after orb-tools update to v12 PR #469

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/circleci-orb#476]
## Issue

- The merge of PR https://github.com/cypress-io/circleci-orb/pull/469 into the `master` branch fails to run the workflow [.circleci/test-deploy.yml](https://github.com/cypress-io/circleci-orb/blob/master/.circleci/test-deploy.yml)

https://app.circleci.com/pipelines/gh/cypress-io/circleci-orb/1945 shows

```text
Error calling workflow: 'test-deploy'
Error calling job: 'orb-tools/publish'
Missing required argument(s): orb_name, vcs_type
```

Renovate has only updated

https://github.com/cypress-io/circleci-orb/blob/bd1d3235778b23914b192a3f99a7f564be42bd1c/.circleci/test-deploy.yml#L4

and has left

https://github.com/cypress-io/circleci-orb/blob/bd1d3235778b23914b192a3f99a7f564be42bd1c/.circleci/config.yml#L4

unchanged.

The [`v12` Migration Guide](https://github.com/CircleCI-Public/orb-tools-orb/blob/master/MIGRATION.md) provides the full set of instructions for migration.

See also https://discuss.circleci.com/t/orb-tools-v12-test-without-publishing/48014

## Suggestion

- revert PR https://github.com/cypress-io/circleci-orb/pull/469

Following the above [`v12` Migration Guide](https://github.com/CircleCI-Public/orb-tools-orb/blob/master/MIGRATION.md) introduces serious breaking changes, since the dash (`-`) character is converted to underscore (`_`) everywhere. It would need to be considered whether a migration to `orb-tools` v12 is necessary and how it should be done to minimize user impact.

**Comment by MikeMcC399** [comment]
- A failed attempt was made through PR #477

**Comment by MikeMcC399** [comment]
After trying out the [migrate.sh](https://github.com/CircleCI-Public/orb-tools-orb/blob/master/migrate.sh) script I realized that this is not a simple update and it is not something that I would want to attempt myself.

The script introduces serious breaking changes, since the dash (`-`) character is converted to underscore (`_`) everywhere (even in places it shouldn't, like for `wait-on`), so if this were to be implemented it would need a v4 release of the `cypress-io/circleci-orb`.

- The best short-term solution is to simply revert PR #469. I have submitted this as PR #478
